<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DialogDriverConflict</class>
 <widget class="QDialog" name="DialogDriverConflict" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>384</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>237</red>
      <green>237</green>
      <blue>237</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
     <color>
      <red>119</red>
      <green>117</green>
      <blue>115</blue>
     </color>
     <color>
      <red>159</red>
      <green>157</green>
      <blue>154</blue>
     </color>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>101</red>
      <green>148</green>
      <blue>235</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
    </active>
    <inactive>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>237</red>
      <green>237</green>
      <blue>237</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
     <color>
      <red>119</red>
      <green>117</green>
      <blue>115</blue>
     </color>
     <color>
      <red>159</red>
      <green>157</green>
      <blue>154</blue>
     </color>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>16</red>
      <green>16</green>
      <blue>16</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>101</red>
      <green>148</green>
      <blue>235</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
    </inactive>
    <disabled>
     <color>
      <red>127</red>
      <green>125</green>
      <blue>123</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>237</red>
      <green>237</green>
      <blue>237</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
     <color>
      <red>119</red>
      <green>117</green>
      <blue>115</blue>
     </color>
     <color>
      <red>159</red>
      <green>157</green>
      <blue>154</blue>
     </color>
     <color>
      <red>127</red>
      <green>125</green>
      <blue>123</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>127</red>
      <green>125</green>
      <blue>123</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>239</red>
      <green>235</green>
      <blue>231</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>84</red>
      <green>123</green>
      <blue>196</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>245</green>
      <blue>243</blue>
     </color>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Driver conflict</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>226</red>
         <green>228</green>
         <blue>252</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>226</red>
         <green>228</green>
         <blue>252</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>84</red>
         <green>123</green>
         <blue>196</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>226</red>
         <green>228</green>
         <blue>252</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>ConfigTool Value</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Icepap Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnStored" >
       <property name="text" >
        <string>Keep stored values</string>
       </property>
       <property name="icon" >
        <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/ipapsys.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnIcepap" >
       <property name="text" >
        <string>Set Icepap values</string>
       </property>
       <property name="icon" >
        <iconset resource="icepapcms.qrc" >:/small_icons/IcepapCfg Icons/Icepapicon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="icepapcms.qrc" />
 </resources>
 <connections/>
</ui>
